docker/build_helper.sh

```diff
@@ -35,7 +35,7 @@ export BUILD_OPENCV=${BUILD_OPENCV:-ON}
 export BUILD_PROTOBUF=${BUILD_PROTOBUF:-ON}
 export BUILD_NVJPEG=${BUILD_NVJPEG:-ON}
 # use a default value as it differs for CUDA 10 and CUDA 11.x
-export BUILD_NVJPEG2K=${BUILD_NVJPEG2K}
+export BUILD_NVJPEG2K=${BUILD_NVJPEG2K:-ON}
```

Comment:
I would check QNX and aarch64 builds for Jetson.

Reply:
QNX fails for an unrelated reason.